### ISSUE TITLE
fails to compile 2.0.9 on alpine

### ISSUE THREAD

**Issue by BertrandGouny** [acassen/keepalived#1060]
Hello,
i can't compile keepalived on alpine : 

```
In file included from rbtree.c:29:0:
rbtree_augmented.h:165:24: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'struct'
 static __always_inline struct rb_node *
                        ^~~~~~
rbtree_augmented.h:273:24: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'void'
 static __always_inline void
                        ^~~~
rbtree_augmented.h:283:24: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'void'
 static __always_inline void
                        ^~~~
rbtree.c:100:24: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'void'
 static __always_inline void
                        ^~~~
rbtree.c:246:24: error: expected '=', ',', ';', 'asm' or '__attribute__' before 'void'
 static __always_inline void
                        ^~~~
```

command used : 
```
apk --no-cache add \
    autoconf \
    curl \
    gcc \
    ipset \
    ipset-dev \
    iptables \
    iptables-dev \
    libnfnetlink \
    libnfnetlink-dev \
    libnl3 \
    libnl3-dev \
    make \
    musl-dev \
    openssl \
    openssl-dev \
    && curl -o keepalived.tar.gz -SL http://keepalived.org/software/keepalived-${KEEPALIVED_VERSION}.tar.gz \
    && mkdir -p /container/keepalived-sources \
    && tar -xzf keepalived.tar.gz --strip 1 -C /container/keepalived-sources \
    && cd container/keepalived-sources \
    && ./configure --disable-dynamic-linking \
    && make && make install \
```

**Comment by pqarmitage** [comment]
Commit 1746fcb should resolve this.

**Comment by pqarmitage** [comment]
I'm about to push a commit that stops the compiler warnings on Alpine in vrrp_daemon.c

**Comment by pqarmitage** [comment]
Commit aa43dba resolves the compiler warnings in vrrp_daemon.c.

I get a <may be used uninitialiszed> in vrrp.c at line 809. Inspecting the code shows that it is initialised, and also later versions of gcc don't produce a warning on this line, so I'm not going to make an unnecessary initialisation just to keep the compiler quiet.

**Comment by BertrandGouny** [comment]
Confirm this is fixed on 2.0.10 thanks a lot.
